tests/unit/query_plan.cpp

```diff
@@ -426,7 +426,54 @@ TYPED_TEST(TestPlanner, CreateWithSum) {
   AstStorage storage;
   auto ident_n = IDENT("n");
   auto n_prop = PROPERTY_LOOKUP(ident_n, prop);
-  auto sum = SUM(n_prop);
+  auto sum = SUM(n_prop, false);
+  auto query = QUERY(SINGLE_QUERY(CREATE(PATTERN(NODE("n"))), WITH(sum, AS("sum"))));
+  auto symbol_table = memgraph::query::MakeSymbolTable(query);
+  auto acc = ExpectAccumulate({symbol_table.at(*ident_n)});
+  auto aggr = ExpectAggregate({sum}, {});
```

Comment:
`ExpectAggregate` doesn't check the value of `distinct`, please update that. You can find it in `memgraph/memgraph/tests/unit/query_plan_checker.hpp`.